blogposts/smart-lock-conclusions.md

```diff
@@ -0,0 +1,67 @@
+
+Despite being very secure, most popular centralized cloud services are a constant target for hackers; users need to blindly trust their security systems and rely entirely on these third party services, which could sell their data or even hide a data breach.
+
+Centralized cloud services are no strangers to major hacks, proving that these security systems are often not enough. There are many examples of IoT devices being compromised or even controlled by malicious attackers. As the number of interconnected devices grows every day, while smart homes and OTA updates become the norm, so does the risk of potential cyber attacks. 
+
+[These are only some of the biggest attacks that happened last year.](https://firedome.io/blog/top-cyber-attacks-on-iot-devices-in-2021/) 
+
+Open source home automation softwares, on the other hand, require maintanance time, backup servers and backup network connectivity. In addition, users would have to open their home or office network to external incoming connections to talk directly to the IoT software that, in turn, controls the devices. This poses security concerns that are not easy to tackle without professional knowledge and a solid security policy. 
+
+Using a smart contract would, instead, be much easier, and much safer from a security standpoint: The device would access the blockchain network as if it were a cloud, avoiding the need to receive incoming connections. Nobody would, furthermore, be able to take control of devices remotely, because they would somehow need to either crack the owner's private key or hack a blockchain (both things deemed not feasible, or almost impossible as of today).
+
+One final benefit of using a public blockchain as a secure IoT cloud, is the fact that there is no limit on the number of *"Things"* that could be controlled.
+
+
+## The Limitations
+
+Blockchain benefits do not come for free: While reading the state of public blockchains is always free, modifying their state requires users to pay a *network fee*. Even when the fee is very low, this model may become expensive, depending on the use case and number of devices. However, when the number of devices is very high and the number of interactions is relatively low, the blockchain *pay-per-use* model may become even cheaper then some centralized services. On the contrary, a high number of interactions might cause potential latency issues. Unlike centralized, cloud-based IoT applications, blockchain is a shared underlying infrastructure for many dApps. As a result, users might experience significant delays when some dApps become very popular and the network gets clogged with a high volume of requests. In the case of our smart-lock application, this would mean that the lock might take several minutes to open!
```

Comment:
I would remove this one because it's not the high number of interactions with the smart lock that leads to latency, rather it could be caused by other smart contracts congesting the network:
>> On the contrary, a high number of interactions might cause potential latency issues.

You can say: "_Using the blockchain to control IoT devices might also cause potential latency issues. Unlike centralized, ..._"

Reply:
Yes, this includes XinXin's comments (sorry I didn't see this message before). I'll make the changes